src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -248,9 +246,6 @@ object ProtocPlugin extends AutoPlugin with Compat {
       if (exitCode != 0)
         sys.error("protoc returned exit code: %d" format exitCode)
 
-      log.info("Compiling protobuf")
-      sortedTargetPaths.foreach { path => log.info("Protoc target: %s".format(path.absolutePath)) }
```

Comment:
this is logged just above before the protoc run, so instead of printing a dedup log, I thought I could simply remove it